proto/hummock.proto

```diff
@@ -40,10 +40,17 @@ message InputLevel {
 }
 
 message LevelDelta {
-  uint32 level_idx = 1;
-  uint64 l0_sub_level_id = 2;
-  repeated uint64 removed_table_ids = 3;
-  repeated SstableInfo inserted_table_infos = 4;
+  enum DeltaType {
+    UNSPECIFIED = 0;
+    GROUPCONSTRUCT = 1;
+    GROUPDESTROY = 2;
+  }
+  DeltaType delta_type = 1;
+  uint32 level_idx = 2;
+  uint64 l0_sub_level_id = 3;
+  repeated uint64 removed_table_ids = 4;
+  repeated SstableInfo inserted_table_infos = 5;
+  CompactionConfig group_config = 6;
```

Comment:
group_config is only used by GROUPCONSTRUCT.
And if we want to make a LevelDelta self-contained, GROUPDESTROY need know group id.
May consider oneof to carry data for DeltaType?

Reply:
> oneof

fixed